test/unit/api/wide/load/logical/load.hpp

```diff
@@ -321,3 +327,61 @@ TTS_CASE_TPL("load for different alignment", EVE_TYPE )
     test(eve::lane<4>);
   }
 }
+
+#if 0
```

Comment:
@jfalcou - can I ask you to try it out?
I think it fails  because of https://github.com/jfalcou/eve/issues/507
fails only on avx512.